modules/seqexec/web/client/src/main/scala/seqexec/web/client/components/HeadersSideBar.scala

```diff
@@ -67,13 +67,20 @@ object HeadersSideBar {
     Show.show(_.label)
 
   @Lenses
-  final case class State(operator: Option[Operator], observer: Option[Observer])
+  final case class State(
+    operator: Option[Operator],
+    observer: Option[Observer],
+    prevOperator: Option[Operator],
```

Comment:
Is this the copy from props? Seems boring to do this but react is forcing us

Reply:
Yes. There's no way anymore to run code at the exact moment that properties change. I've only copied the ones we seem to need and prefixed them with `prev`. Maybe in some cases there's a way to avoid this by doing some more refactoring, but I tried to make everything work by keeping the existing logic for now.